### PR TITLE
APIGOV-24529 - Add new API Gateway environment variable for TA

### PR DESCRIPTION
**Pull request by jcollins-axway**
## Describe the changes

Add a new environment variable used to count the usage of specified services

## Checklist for contributors

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

### CODE REVIEW

**Comment by jcollins-axway** [comment]
@sbolosan @vivekschauhan - any feedback?

**Review by sbolosan — APPROVED** [review]
Sorry, saw this 🤦‍♂️

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | 3b9c38c |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-264.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/connect_manage_environ/connect_api_manager/agent-variables.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ You can extend the previous minimum variable list with the following variables.
...
 | **Amplify Central variables**                                      |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             |
 | CENTRAL_DEPLOYMENT                                                 | Specifies region (default: US = `prod` / EU = `prod-eu`).                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
Comment (robertengaxway) [inline content/en/docs/connect_manage_environ/connect_api_manager/agent-variables.md:103]: should we add something for APAC region support along the lines of:
AP ='prod-ap'

**Review by sbolosan — APPROVED** [review]
that's easy to miss on a review

**Review by lbadenhop — APPROVED** [review]

**Review by robertengaxway — COMMENTED** [review]
Approved

**Pull request merged**